src/Propel/Generator/Builder/Om/ObjectBuilder.php

```diff
@@ -898,7 +898,7 @@ protected function addTemporalAccessorBody(&$script, Column $column)
 
         $script .= "
         if (\$format === null) {
-            return \$this->$clo;
+            return \$this->$clo instanceof \DateTime ? clone \$this->{$clo} : null;
```

Comment:
Hm, makes sense. I assumed $this->{$clo} will always be either DateTime or null, but you're probably right, I can adjust the commit when we have a consensus of what's the best way to proceed in #949.